### ISSUE TITLE
view_multiple does not refresh collection contents properly

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#5060]
after reload:
![screenshot 2017-11-22 15 35 39](https://user-images.githubusercontent.com/1814954/33148719-e0932a68-cf9a-11e7-81e5-ee6778770722.png)

after 'close and open' of the collection
![screenshot 2017-11-22 15 35 45](https://user-images.githubusercontent.com/1814954/33148720-e0a53938-cf9a-11e7-8cb9-028de9291078.png)

**Comment by mvdbeek** [comment]
Yeah, that's stochastic error that has been there since the initial implementation, I think. Would be great to fix this.
